communication/ws.go

```diff
@@ -58,10 +58,13 @@ func wsEndpoint(w http.ResponseWriter, r *http.Request) {
 
 	log.Println(player.Name + " has connected")
 
+	updateRocketChat(lobby, player)
+
 	player.SetWebsocket(ws)
 	game.OnConnected(lobby, player)
 
 	ws.SetCloseHandler(func(code int, text string) error {
+		updateRocketChat(lobby, player)
 		game.OnDisconnected(lobby, player)
```

Comment:
Maybe this should be in `OnDisconnected` itself, since there's a situation where the close-handler doesn't get called due to a bug in a dependency. Just like the call before, this should run in a goroutine. Maybe the routine should just be launched in the `updateRocketChat` method.

Reply:
Yes, I thought about adding this in `OnConnected` and `OnDisconnected`, but then I would have to reference the `communication` package. Where do you propose I add my own code? I could make a new file in the `game` package, but it seems a bit wrong.

Reply:
I see.  It's fine for now then.